erpnext/stock/doctype/stock_entry/stock_entry.js

```diff
@@ -115,6 +115,13 @@ frappe.ui.form.on('Stock Entry', {
 			return;
 		}
 
+		if (!frm.is_new() && frm.doc.docstatus === 0) {
+			frm.add_custom_button(__("Quality Inspection(s)"), () => {
+				frm.trigger("make_quality_inspection");
```

Comment:
reuse the method in `transaction.js`
```suggestion
				let transaction_controller = new erpnext.TransactionController({frm: frm});
				transaction_controller.make_quality_inspection();
```